recipes/lytekit/meta.yaml

```
@@ -0,0 +1,90 @@
+{% set name = "lytekit" %}
+{% set version = "0.15.27" %}
+{% set python_max = "3.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lytekit-{{ version }}.tar.gz
+  sha256: bf3d9e127ad02b94e88c9b7f4f8df2e81946104c470c935d00ad99b312f46915
+
+build:
+  skip: true  # [py<37 or py>310]
```

I could not get it to work with requiring a python version lower than 3.11 AND having it be no-arch.  Suggestions welcome, but I think this is ready.

---

The only error was this in **win-64** build

```python
Traceback (most recent call last):
  File "D:\bld\lytekit_1741026804461\_test_env\Scripts\flyte-cli-script.py", line 5, in <module>
    from flytekit.clis.flyte_cli.main import _flyte_cli
  File "D:\bld\lytekit_1741026804461\_test_env\lib\site-packages\flytekit\clis\flyte_cli\main.py", line 19, in <module>
    from google.protobuf.pyext.cpp_message import GeneratedProtocolMessageType as _GeneratedProtocolMessageType
  File "D:\bld\lytekit_1741026804461\_test_env\lib\site-packages\google\protobuf\pyext\cpp_message.py", line 39, in <module>
    from google.protobuf.pyext import _message
ImportError: cannot import name '_message' from 'google.protobuf.pyext' (D:\bld\lytekit_1741026804461\_test_env\lib\site-packages\google\protobuf\pyext\__init__.py)
```

Since only **linux-64** actually needs to build for **noarch** recipes, and this seem like specifically an internal problem with `protobuf` **win-64** builds, I'd suggest that the error can be ignored and this should be re-reviewed *after converting back to **noarch***.

---

> _ after converting back to noarch._

I’ve tried a bunch of times to make it “no arch”but the Python selectors preclude this. I’d welcome some help given two folks have suggested this, so I’m missing something obvious 😅